src/data/metadata.csv

```diff
@@ -8243,14 +8243,54 @@ LT,LT,Lithuania,,,,,,,,-1
 LU,LU,Luxembourg,,,,,,,,-1
 LV,LV,Latvia,,,,,,,,-1
 LY,LY,Libya,,,,,,,,-1
-LY_BA,LY,Libya,BA,Benghazi,,,,,Benghazi,
-LY_JI,LY,Libya,JI,Jafara,,,,,Al-Jafara,
-LY_MS,LY,Libya,MS,Misrata District,,,,,Misrata,
-LY_SB,LY,Libya,SB,Sabha District,,,,,Sabha,
-LY_ST,LY,Libya,ST,Sabratha District,,,,,Sabratha,
-LY_TB,LY,Libya,TB,Tripoli District,,,,,Tripoli,
-LY_WS,LY,Libya,WS,Wadi al Shatii District,,,,,Wadi al Shati,
-LY_ZL,LY,Libya,ZL,Zliten District,,,,,Zliten,
+LY_BA,LY,Libya,,,BA,Benghazi,,,,
+LY_BU,LY,Libya,,,BU,Butnan,,,,
+LY_BW,LY,Libya,,,BW,Bani Walid,,,,
+LY_DR,LY,Libya,,,DR,Derna,,,,
+LY_GD,LY,Libya,,,GD,Ghadames,,,,
+LY_GH,LY,Libya,,,GH,Gharyan,,,,
+LY_GT,LY,Libya,,,GT,Ghat,,,,
+LY_JA,LY,Libya,,,JA,Jabal al Akhdar,,,,
+LY_JA_BY,LY,Libya,JA,Jabal al Akhdar,BY,Al Bayda,,,,
+LY_JA_QA,LY,Libya,JA,Jabal al Akhdar,QA,Qasr Al-Akhyar,,,,
+LY_JG,LY,Libya,,,JG,Jabal al Gharbi,,,,
+LY_JG_KK,LY,Libya,JG,Jabal al Gharbi,KK,kikla,,,,
+LY_JG_ZI,LY,Libya,JG,Jabal al Gharbi,ZI,Zliten,,,,
+LY_JI,LY,Libya,,,JI,Al-Jafra,,,,
+LY_JU,LY,Libya,,,JU,Al Jufrah,,,,
+LY_JU_SK,LY,Libya,JU,Al Jufrah,SK,Sawknah,,,,
```

Comment:
I'm confused by how these are laid out. If LY_JU_SK is within LY_JU, then LY_JU should be admin level 1:
```
LY_JU,LY,Libya,JU,Al Jufrah,,,,,,
LY_JU_SK,LY,Libya,JU,Al Jufrah,SK,Sawknah,,,,
```

Then you would need to change the field being output by the script back to `match_string` but, as long as no two places are named the same within Libya, the regions should be appropriately matched since `match_string` will attempt to match against either `subegion2_name` or `subregion1_name`.

Reply:
Got it, fixed the ordering.